readthedocs_theme/templates/includes/topnav.html

```diff
@@ -135,7 +131,7 @@
 
               <div class="item">
                 {# TODO: Add support page to marketing website #}
-                <div class="ui wide circular basic icon dropdown button" data-module="dropdown" data-action="select" data-display-type="block">
+                <div class="ui circular basic icon wide pointing top right dropdown button" data-module="dropdown" data-action="select" data-display-type="block">
```

Comment:
I had tried `pointing` previously, but didn't realize it required additional classes. I find it really hard to figure out what combination of classes work, and it also seems like the order matters, which I've also struggled with. But it's always obvious once there's an example, just very hard to actually get a working example for me.

Reply:
Aye, ordering *can* be important for some of the rules, though most are okay in any order. There are some directional classes like `top right` that I think might be order specific. I do try keep classes in some order of ascending importance, but this is very subjective and not a firm rule.

Pointing menu directions aren't covered super well in the docs that I can find. The variant is covered here, and does mention the extra classes in the examples, but does not do a good job of calling out how to use them:
https://fomantic-ui.com/modules/dropdown.html#pointing

I feel like this is a gap in the docs and I probably know this from working with another similar element like popups or something.